internal/cli/atlas/deployments/setup.go

```diff
@@ -94,7 +96,7 @@ func (opts *SetupOpts) configureMongod(podmanOpts podman.Client) error {
 	keyfile := "/data/configdb/keyfile"
 	keyfilePerm := 400
 
-	createKeyfile := fmt.Sprintf("echo '%s' > %s", opts.deploymentName, keyfile)
+	createKeyfile := fmt.Sprintf("echo '%s' > %s", opts.deploymentID, keyfile)
```

Comment:
how about sending the UUID here instead? I don't think `deploymentID` is very useful to be extracted outside at the moment

Reply:
```suggestion
        key := uuid.NewString()
	createKeyfile := fmt.Sprintf("echo '%s' > %s", key, keyfile)
```